misc/scripts/header_guards.py

```diff
@@ -32,138 +31,53 @@
                 break
         else:
             if not sline.startswith("*") and not sline.startswith("/*"):  # Not in the Godot header anymore.
-                HEADER_CHECK_OFFSET = idx + 1  # The include should be two lines below the Godot header.
+                HEADER_CHECK_OFFSET = idx + 1  # The guard should be two lines below the Godot header.
                 break
 
     if HEADER_CHECK_OFFSET < 0:
```

Comment:
If the file ends right after the the header, then `lines[HEADER_CHECK_OFFSET]` will raise an exception
```suggestion
    if HEADER_CHECK_OFFSET < 0 or HEADER_CHECK_OFFSET >= len(lines):
```